tzlocal/darwin.py

```diff
@@ -11,7 +12,8 @@ def _get_localzone():
         "systemsetup -gettimezone",
         shell=True,
         stderr=subprocess.PIPE,
-        stdout=subprocess.PIPE
+        stdout=subprocess.PIPE,
+        universal_newlines=True,
```

Comment:
This doesn't change it to unicode, but will convert newlines. I guess it doesn't hurt, but what is the problem it solves, really?